docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,334 @@
+---
+layout: page
+title: ksqlDB with Embedded Connect
+tagline: Run Kafka Connect embedded within ksqlDB
+description: Learn how to use ksqlDB with embedded Connect to integrate with external data sources and sinks
+keywords: ksqlDB, connect, PostgreSQL, jdbc
+---
+
+Overview
+==============
+
+This tutorial will demonstrate how to integrate ksqlDB with an external data source to power a simple ride sharing app. Our external source will be a PostgreSQL database containing relatively static data describing each driver’s vehicle. By combining this human-friendly static data with a continuous stream of computer-friendly driver and rider location events, we derive an enriched output stream that the ride sharing app may use to facilitate a rendezvous in real time.
```

Comment:
```suggestion
This tutorial shows how to integrate ksqlDB with an external data source to power a simple ridesharing app. The external source is a PostgreSQL database containing relatively static data that describes each driver’s vehicle. This tutorial combines the human-friendly static data with a continuous stream of computer-friendly driver and rider location events to derive an enriched output stream that the ridesharing app uses to set up a rendezvous in real time.
```

Reply:
Can you please break these paras at column 80?

Reply:
My bad, I will ensure all lines are broken at 80 chars 👍